Source/Widgets/Viewer/Viewer.js

```diff
@@ -1359,12 +1357,12 @@ Either specify options.terrainProvider instead or set options.baseLayerPicker to
         var animationExists = defined(this._animation);
         var timelineExists = defined(this._timeline);
 
-        if (!this._forceResize && width === this._lastWidth && height === this._lastHeight) {
+        cesiumWidget.resize();
```

Comment:
Why was forceResize removed and the logic changed here?  It maybe the the right thing to do, but your PR description should have mentioned that this changes and the reasoning for doing so.

Reply:
@mramato updated